docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -25,22 +25,100 @@ under the License.
 
 # Elastic Scaling
 
-Apache Flink allows you to rescale your jobs. You can do this manually by stopping the job and restarting from the savepoint created during shutdown with a different parallelism.
+Apache Flink allows you to rescale your jobs when the job is either not-catching up with inputs or over-provisioned. You can do this by manually stopping the job and restarting from the savepoint created during shutdown with a different parallelism.
 
-This page describes options where Flink automatically adjusts the parallelism instead.
+This page describes options **where Flink automatically adjusts the parallelism** instead, without relying on external orchestration.
```

Comment:
The previous versions wasn't great either, but combined with the above statement this sounds very much like autoscaling which this obviously isn't. (Although it maybe is for the adaptive batch scheduler).

It also conflicts a bit with EDRM which is all about external orchestration.

I think this whole section needs a rewrite.

```
The parallelism of a job has traditionally been a constant that is fixed when the job was submitted.
Batch jobs couldn't be rescaled at all,
while Streaming jobs could be stopped with a savepoint and restarted with a different parallelism.

This page describes features that allow the parallelism of a job to change at runtime.
```